### DISCUSSION TITLE
oVirt 4.5.0 (End Of Life) ? On-going Support

### DISCUSSION THREAD

**Discussion by masoodahmed100** [oVirt/ovirt-site#3149] [General]
Hi,
I am working on a project and we would like to utilise (oVirt 4.5.) as the platform for our enterprise infrastructure.
We have arrived at this juncture because of a range of considerations related to hardware and software availability and compatibility.

My question is if there is on-going vendor \ professional support for oVirt moving forward, as one of my colleagues spoke with Red Hat, who stated they are going to end their support for the product.

We will not gain security accreditation for the product unless we have professional support in place, and cannot rely on just community support.

Thanks

**Comment by michalskrivanek** [comment]
Hey,
Red Hat's RHV entered Maintenance last year (see https://access.redhat.com/support/policy/updates/rhev) with transitioning virtualization usecases to Openshift Virtualization (part of Openshift Container Platform product)
Oracle has commercial support for OLVM, I don't know their plans/support scope.
There are also other smaller/local vendors supporting oVirt (either directly or "customized"), but we do not keep any list of them.

**Comment by masoodahmed100** [comment]
Hi Michael,

Thanks for the prompt response.

We have considered other products, but are leaning towards oVirt due to a
number of logistical issues, where we cannot procure more hardware due to
timescales and need a Platform solution Asap.

In relation to OLVM, is that a separate product that is compatible with
oVirt? or are you suggesting an alternative to oVirt that may have on-going
vendor support?

Thanks

On Fri, 21 Apr 2023 at 10:16, michalskrivanek ***@***.***>
wrote:

> Hey,
> Red Hat's RHV entered Maintenance last year (see
> https://access.redhat.com/support/policy/updates/rhev) with transitioning
> virtualization usecases to Openshift Virtualization (part of Openshift
> Container Platform product)
> Oracle has commercial support for OLVM, I don't know their plans/support
> scope.
> There are also other smaller/local vendors supporting oVirt (either
> directly or "customized"), but we do not keep any list of them.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/oVirt/discussions/3149#discussioncomment-5684078>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7HPO2VL45QJYYWHEUKPRY3XCJGAVANCNFSM6AAAAAAXGTLPDE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Regards
Masood Ahmed

07813018214

**Reply by michalskrivanek** [reply]
OLVM is similarly to RHV a product derived from oVirt. The oVirt contributors are almost exclusively from Red Hat so RHV is/was the primary "target" for development, and with my Red Hat hat on (ha!:) I would recommend looking at Openshift Virtualization as it gives you not only classical virtualization(underneath it runs the same QEMU/KVM as oVirt) but also enterprise kubernetes platform for containers. 
But hardware constraints might justify something more simple like oVirt, then it depends what kind of engagement you really want, RHV is not sold anymore(I think, at least. Even if it is the end of maintenance is next year so it's not a sensible choice), OLVM might be a better choice (but I don't know the details, pricing, etc), and smaller vendors might exist, but from the oVirt development POV there's not much going on besides user-to-user help on mailing list.
I hope that helps...

**Comment by masoodahmed100** [comment]
Thanks, much appreciated.

On Fri, 21 Apr 2023 at 12:14, michalskrivanek ***@***.***>
wrote:

> OLVM is similarly to RHV a product derived from oVirt. The oVirt
> contributors are almost exclusively from Red Hat so RHV is/was the primary
> "target" for development, and with my Red Hat hat on (ha!:) I would
> recommend looking at Openshift Virtualization as it gives you not only
> classical virtualization(underneath it runs the same QEMU/KVM as oVirt) but
> also enterprise kubernetes platform for containers.
> But hardware constraints might justify something more simple like oVirt,
> then it depends what kind of engagement you really want, RHV is not sold
> anymore(I think, at least. Even if it is the end of maintenance is next
> year so it's not a sensible choice), OLVM might be a better choice (but I
> don't know the details, pricing, etc), and smaller vendors might exist, but
> from the oVirt development POV there's not much going on besides
> user-to-user help on mailing list.
> I hope that helps...
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/oVirt/discussions/3149#discussioncomment-5685224>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7HPO2XQINJIBGZWPDKWEWTXCJTYZANCNFSM6AAAAAAXGTLPDE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Regards
Masood Ahmed

07813018214
